src/bundle/Resources/config/views.yaml

```diff
@@ -0,0 +1,38 @@
+services:
+    Ibexa\Platform\Search\View\SearchViewProvider:
+        arguments:
+            $matcherFactory: '@platform.user.view.search.matcher_factory.dynamically_configured'
+        tags:
+            - { name: ezpublish.view_provider, type: Ibexa\Platform\Search\View\SearchView, priority: 10 }
+
+    platform.view.search.matcher_factory:
```

Comment:
not entirely sold on this one, because `platform` w/o context feels too generic. Since we're prefixing everything as Ibexa\Platform, we probably should use `ibexa.platform` (though should you use 'ezplatform` I wouldn't mind it)